scripts/serverless.js

```diff
@@ -495,24 +497,38 @@ const processSpanPromise = (async () => {
 
     const configurationFilename = configuration && configurationPath.slice(serviceDir.length + 1);
 
-    if (isInteractiveSetup) {
-      require('../lib/cli/ensure-supported-command')(configuration);
+    const isStandaloneCommand = standaloneCommands.includes(command);
 
-      if (!process.stdin.isTTY && !process.env.SLS_INTERACTIVE_SETUP_ENABLE) {
-        throw new ServerlessError(
-          'Attempted to run an interactive setup in non TTY environment.\n' +
-            "If that's intentended enforce with SLS_INTERACTIVE_SETUP_ENABLE=1 environment variable",
-          'INTERACTIVE_SETUP_IN_NON_TTY'
-        );
-      }
-      const { configuration: configurationFromInteractive } =
-        await require('../lib/cli/interactive-setup')({
+    if (isInteractiveSetup || isStandaloneCommand) {
+      let configurationFromInteractive;
```

Comment:
I think we can reuse `configuration` variable, and simply override it in case of interactive CLI as:

```javascript
if (result.configuration) configuration = result.configuration
```

Reply:
Cleaner! Thanks to let me know I can override that. Was tempted